### DISCUSSION TITLE
Images don't appeared in exported html

### DISCUSSION THREAD

**Discussion by gitcat87** [marp-team/marp#421] [Q&A] [answered]
When I previews slides in VSCode and when I exported them to HTML, the images are not shown.
It comes like this:
![noname](https://user-images.githubusercontent.com/120068782/222625136-85b32149-0055-4077-855b-cdeb625cdb5d.png)

What I used the code was
`![w:1000 h:450 center](1.png)`
when I exported in PDF, PPTX, it goes well. only has a problem in html

**Comment by yhatt** [comment, marked as answer]
Didn't you export HTML into a different directory than the directory that that Markdown is located? Try to export HTML into the same directory as Markdown.

Every path to refer slide resources will respect original Markdown. If the directory structure was changed from Markdown, the browser may become not resolvable the location of images.

**Reply by gitcat87** [reply]
It worked on my computer, but when I execute it others, it didn't go well. What else I missed?
